### ISSUE TITLE
Additional Scaling Options do not appear to be working

### ISSUE THREAD

**Issue by S1L3NTW0LF** [Toxicom/toxiui#154]
### Description
Additional Scaling Options not consistently working

Scaling options only appear to take effect when the values are edited.  On game load or any in-game reload, all scaling options return to 1.0 regardless of the values set.  Video demonstrates exact behavior for clarity.

### Lua error
no errors

### ToxiUI Status Report
![image](https://github.com/user-attachments/assets/a8f67047-d40c-435d-a710-8475648af085)

### Screenshots
Screenshots often say more than words

![image](https://github.com/user-attachments/assets/7f64ce47-451b-4ea1-bb36-8e91ed293d72)

Video capture to demonstrate
https://youtu.be/_OwZw7RQDKk

### To reproduce
If this issue is describing a bug, include some steps to reproduce the behavior

1.  ElvUI > ToxiUI > Plugins > Additional Scaling > set Character Frame to 1.25
2.  /reload
3.  Character frame returns to 1.0, even though the value is still set to 1.25

**Comment by Toxicom** [comment]
Thanks for the report and the video, it helped!

Are you able to reproduce this with `/tx debug on`?

**Comment by S1L3NTW0LF** [comment]
You're welcome!   Glad the video is helpful.   

Looks like scaling works fine with debug enabled.   I checked Character Frame, Group Finder, Collections...  all seem to work fine when debug is on.

**Comment by Toxicom** [comment]
Means some other addon is overriding the scaling. Not much we can do there unless you figure out which one exactly and maybe then we can do something about it

**Comment by S1L3NTW0LF** [comment]
Ahh thanks @Toxicom makes sense!!

After your update, I was able to find the problem AddOn.  I was using DeModal to allow me to move frames that aren't normally moveable.  I disabled that and the problem went away.   Looks like WindTools has an option for moving frames anyways, just tested and that seems to work fine.  So I should be able to remove that addon.   

Problem solved, appreciate your time!
